test/pylib/random_tables.py

```diff
@@ -182,8 +182,9 @@ async def add_column(self, name: str = None, ctype: Type[ValueType] = None, colu
             column = Column(name, ctype=ctype)
         self.columns.append(column)
         assert self.manager.cql is not None
-        await self.manager.cql.run_async(f"ALTER TABLE {self.full_name} "
```

Comment:
The commit message says:

> When adding a new column it's always a value column, not clustering. Fix
the name and sequence.

no such thing is happening in this commit though.

Reply:
Yes, another commit was in master. I removed the commit in this series and rebased. Please latest.